ui/app/routes/vault/cluster/access/mfa/index.js

```diff
@@ -0,0 +1,37 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+export default class MfaRoute extends Route {
+  @service router;
+
+  queryParams = {
+    page: {
+      refreshModel: true,
+    },
+  };
+
+  model(params) {
+    return this.store
+      .lazyPaginatedQuery('mfa-method', {
+        responsePath: 'data.keys',
```

Comment:
I'm not sure we want to use the `keys` in this case since we have all the data returned under `data.key_info`. Simply updating the `responsePath` might not work though since `key_info` is an object rather than an array. In the query method on the adapter it might be best to mutate the response and loop through the `key_info` objects to turn it into an array.

Reply:
Yup responsePath cannot use key_info. Instead of handling this in adapter, I have updated the serializer to normalize the response based on keys and key_info.